t/14-grutasks.t

```diff
@@ -241,8 +249,8 @@ is(
 $assets->update({size => 26 * $gib});
 run_gru_job($t->app, 'limit_assets');
 
-is(scalar @{mock_removed()}, 1, "one asset should have been 'removed' at size 26GiB");
-is(scalar @{mock_deleted()}, 1, "one asset should have been 'deleted' at size 26GiB");
+is(scalar @{mock_removed()}, 1, "assets should have been 'removed' at size 26GiB");
+is(scalar @{mock_deleted()}, 1, "assets should have been 'deleted' at size 26GiB");
```

Comment:
I don't really care but would personally prefer the singular. We could also simplify the phrasing:
```suggestion
is(scalar @{mock_removed()}, 1, "asset is 'removed' at size 26GiB");
is(scalar @{mock_deleted()}, 1, "asset is 'deleted' at size 26GiB");
```